fn/slice.go

```diff
@@ -17,7 +17,7 @@ type Number interface {
 
 // All returns true when the supplied predicate evaluates to true for all of
 // the values in the slice.
-func All[A any](pred func(A) bool, s []A) bool {
+func All[A any](s []A, pred Pred[A]) bool {
```

Comment:
IIUC this change is needed, because the predicate is often a closure and it is more convenient to pass it as last argument. Please add the reason to have the change to the commit message.

Reply:
Yes we very often define the functional arguments on the fly making them read better as the last argument.